### PR TITLE
No input type name in method name

### PR DESCRIPTION
**Pull request by garydgregory**

### CODE REVIEW

**Comment by garydgregory** [comment]
Sure but they'll get squashed on merge.

**Comment by michael-o** [comment]
Alright then.

**Comment by garydgregory** [comment]
I squashed 'em here anyway.

File: httpclient5/src/main/java/org/apache/hc/client5/http/async/methods/SimpleHttpRequest.java [hunk 1]
```diff
@@ -85,11 +85,11 @@ public void setBody(final SimpleBody body) {
         this.body = body;
     }
 
-    public void setBodyBytes(final byte[] bodyBytes, final ContentType contentType) {
+    public void setBody(final byte[] bodyBytes, final ContentType contentType) {
```
Comment (ok2c) [inline httpclient5/src/main/java/org/apache/hc/client5/http/async/methods/SimpleHttpRequest.java]: @garydgregory Please keep the original methods. Just deprecate them.

Comment (garydgregory) [reply httpclient5/src/main/java/org/apache/hc/client5/http/async/methods/SimpleHttpRequest.java]: Done.

**Comment by ok2c** [comment]
@garydgregory Looks good. Please rebase off the latest master. There is an unrelated commit in the commit list.

**Pull request closed without merging**